JSQMessagesViewController/Model/JSQVideoMediaItem.m

```diff
@@ -101,6 +105,22 @@ - (NSUInteger)mediaHash
     return self.hash;
 }
 
+#pragma mark - Private helper methods
+
+- (void)configureVideoViewWithDefaultPlayIcon
+{
+    UIImageView *imageView = [JSQMessagesVideoThumbnailFactory defaultPlayIcon];
+    [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:imageView isOutgoing:self.appliesMediaViewMaskAsOutgoing];
+    self.cachedVideoImageView = imageView;
+}
+
+- (void)configureVideoViewWithCustomThumbnail
+{
+    UIImageView *imageView = [JSQMessagesVideoThumbnailFactory defaultPlayIconWithThumbnailImage:self.thumbnailImage];
+    [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:imageView isOutgoing:self.appliesMediaViewMaskAsOutgoing];
+    self.cachedVideoImageView = imageView;
+}
```

Comment:
let's remove these methods.

we don't want this class to know about `JSQMessagesVideoThumbnailFactory`.